### ISSUE TITLE
FATAL UNHANDLED EXCEPTION: System.TypeLoadException: Could not resolve type with token 01000008

### ISSUE THREAD

**Issue by basictheprogram** [tModLoader/tModLoader#1152]
### Description

[ERROR] FATAL UNHANDLED EXCEPTION: System.TypeLoadException: Could not resolve type with token 01000008

### Log file

```
[19:40:02] [1/INFO] [tML]: Starting tModLoader v0.11.7.7 Linux server (10/18/2020)
[19:40:02] [1/INFO] [tML]: Running on Mono 4.6.1
[19:40:02] [1/INFO] [tML]: Executable: /var/tModloader/tModLoaderServer.exe
[19:40:02] [1/INFO] [tML]: Working Directory: /var/tModloader
[19:40:02] [1/INFO] [tML]: Launch Parameters:
[19:40:02] [1/WARN] [tML]: First-chance exception reporting is not implemented on Mono
[19:40:02] [1/DEBUG] [tML]: Assembly Resolve: Terraria, Version=1.3.5.3, Culture=neutral, PublicKeyToken=null -> MonoMod.RuntimeDetour, Version=20.5.14.1, Culture=neutral, PublicKeyToken=null
[19:40:02] [1/DEBUG] [tML]: Assembly Resolve: Terraria, Version=1.3.5.3, Culture=neutral, PublicKeyToken=null -> MonoMod.Utils, Version=20.5.14.1, Culture=neutral, PublicKeyToken=null
[19:40:02] [1/DEBUG] [tML]: Assembly Resolve: MonoMod.Utils, Version=20.5.14.1, Culture=neutral, PublicKeyToken=null -> Mono.Cecil, Version=0.11.2.0, Culture=neutral, PublicKeyToken=50cebf1cceb9d05e
[19:40:02] [1/DEBUG] [tML]: Assembly Resolve: Terraria, Version=1.3.5.3, Culture=neutral, PublicKeyToken=null -> Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed
[19:40:02] [1/DEBUG] [tML]: Assembly Resolve: Terraria, Version=1.3.5.3, Culture=neutral, PublicKeyToken=null -> Ionic.Zip.Reduced, Version=1.9.1.8, Culture=neutral, PublicKeyToken=edbe51ad942a3f5c
[19:40:02] [1/DEBUG] [tML]: Hook IL.Ionic.Zip.ZipEntry::ValidateOutput(string, Stream, out string) modified by Terraria
[19:40:05] [1/INFO] [tML]: Unloading mods
[19:40:05] [1/INFO] [tML]: Finding Mods...
[19:40:09] [1/INFO] [tML]: Sandboxing: AnglerQuestAnnouncement
[19:40:09] [1/DEBUG] [tML]: Assembly Resolve: Terraria, Version=1.3.5.3, Culture=neutral, PublicKeyToken=null -> Mono.Cecil.Mdb, Version=0.11.2.0, Culture=neutral, PublicKeyToken=50cebf1cceb9d05e
[19:40:09] [1/INFO] [tML]: Sandboxing: BossChecklist
[19:40:09] [1/INFO] [tML]: Sandboxing: CalamityMod
[19:40:11] [1/INFO] [tML]: Sandboxing: CalamityEnglish
[19:40:11] [1/INFO] [tML]: Sandboxing: CalamityModMusic
[19:40:11] [1/INFO] [tML]: Sandboxing: CalTEDestroyer
[19:40:11] [1/INFO] [tML]: Sandboxing: CalValEX
[19:40:12] [1/INFO] [tML]: Sandboxing: CMusicPlanet
[19:40:12] [1/INFO] [tML]: Sandboxing: MagicStorage
[19:40:12] [1/INFO] [tML]: Sandboxing: RecipeBrowser
[19:40:12] [1/INFO] [tML]: Instantiating Mods...
[19:40:13] [1/DEBUG] [tML]: Assembly Resolve: MagicStorage_0, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null -> Terraria, Version=1.3.5.1, Culture=neutral, PublicKeyToken=null
[19:40:13] [1/DEBUG] [tML]: Assembly Resolve: RecipeBrowser_0, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null -> Terraria, Version=1.3.5.1, Culture=neutral, PublicKeyToken=null
[19:40:13] [1/INFO] [tML]: Initializing: ModLoader v0.11.7.7
[19:40:13] [1/INFO] [tML]: Initializing: AnglerQuestAnnouncement v3.0.0.1
[19:40:13] [1/DEBUG] [tML]: Assembly Resolve: AnglerQuestAnnouncement_0, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null -> TerrariaHooks, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null
[19:40:13] [1/DEBUG] [tML]: Assembly Resolve: TerrariaHooks, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null -> Terraria, Version=1.3.5.1, Culture=neutral, PublicKeyToken=null
[19:40:13] [1/DEBUG] [tML]: Assembly Resolve: TerrariaHooks, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null -> MonoMod.RuntimeDetour, Version=19.12.19.1, Culture=neutral, PublicKeyToken=null
[19:40:13] [1/WARN] [tML]: Upgraded Reference MonoMod.RuntimeDetour -> Version=19.12.19.1 -> 20.5.14.1
[19:40:13] [1/DEBUG] [tML]: Hook On.Terraria.Main::AnglerQuestSwap() added by AnglerQuestAnnouncement_0
[19:40:13] [1/DEBUG] [tML]: Hook On.Terraria.Main::UpdateTime() added by AnglerQuestAnnouncement_0
[19:40:14] [1/INFO] [tML]: Initializing: BossChecklist v1.1.4.1
[19:40:14] [1/DEBUG] [tML]: Hook On.Terraria.GameContent.Events.DD2Event::WinInvasionInternal() added by BossChecklist_0
[19:40:14] [1/INFO] [BossChecklist]: Boss Log integration messages will not be logged.
[19:40:14] [1/INFO] [tML]: Initializing: CalamityMod v1.4.5.3
[19:40:15] [1/DEBUG] [tML]: Assembly Resolve: TerrariaHooks, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null -> MonoMod.Utils, Version=19.12.19.1, Culture=neutral, PublicKeyToken=null
[19:40:15] [1/WARN] [tML]: Upgraded Reference MonoMod.Utils -> Version=19.12.19.1 -> 20.5.14.1
[19:40:15] [1/DEBUG] [tML]: Hook IL.Terraria.Wiring::HitWireSingle(int, int) modified by CalamityMod_0
[19:40:15] [1/DEBUG] [tML]: Hook On.Terraria.Main::InitLifeBytes() added by CalamityMod_0
[19:40:15] [1/DEBUG] [tML]: Hook IL.Terraria.WorldGen::MakeDungeon(int, int) modified by CalamityMod_0
[19:40:16] [1/DEBUG] [tML]: Hook On.Terraria.Player::TileInteractionsUse(int, int) added by CalamityMod_0
[19:40:16] [1/DEBUG] [tML]: Hook On.Terraria.WorldGen::OpenDoor(int, int, int) added by CalamityMod_0
[19:40:16] [1/DEBUG] [tML]: Hook On.Terraria.WorldGen::CloseDoor(int, int, bool) added by CalamityMod_0
[19:40:16] [1/INFO] [tML]: Initializing: CalamityEnglish v0.3
[19:40:16] [1/INFO] [tML]: Initializing: CalamityModMusic v1.4.5.3
[19:40:16] [1/INFO] [tML]: Initializing: CalTEDestroyer v1.0
[19:40:17] [1/INFO] [tML]: Initializing: CalValEX v6.0
[19:40:17] [1/DEBUG] [tML]: Hook IL.Terraria.Wiring::HitWireSingle(int, int) modified by CalValEX_0
[19:40:17] [1/DEBUG] [tML]: Hook IL.Terraria.Wiring::HitWireSingle(int, int) modified by CalValEX_0
[19:40:17] [1/DEBUG] [tML]: Hook IL.Terraria.Wiring::HitWireSingle(int, int) modified by CalValEX_0
[19:40:17] [1/DEBUG] [tML]: Hook IL.Terraria.Wiring::HitWireSingle(int, int) modified by CalValEX_0
[19:40:17] [1/DEBUG] [tML]: Hook IL.Terraria.Wiring::HitWireSingle(int, int) modified by CalValEX_0
[19:40:17] [1/DEBUG] [tML]: Hook IL.Terraria.Wiring::HitWireSingle(int, int) modified by CalValEX_0
[19:40:18] [1/DEBUG] [tML]: Hook IL.Terraria.Wiring::HitWireSingle(int, int) modified by CalValEX_0
[19:40:18] [1/DEBUG] [tML]: Hook IL.Terraria.Wiring::HitWireSingle(int, int) modified by CalValEX_0
[19:40:18] [1/DEBUG] [tML]: Hook IL.Terraria.Wiring::HitWireSingle(int, int) modified by CalValEX_0
[19:40:18] [1/DEBUG] [tML]: Hook IL.Terraria.Wiring::HitWireSingle(int, int) modified by CalValEX_0
[19:40:18] [1/DEBUG] [tML]: Hook IL.Terraria.Wiring::HitWireSingle(int, int) modified by CalValEX_0
[19:40:19] [1/DEBUG] [tML]: Hook IL.Terraria.Wiring::HitWireSingle(int, int) modified by CalValEX_0
[19:40:19] [1/INFO] [tML]: Initializing: CMusicPlanet v1.1
[19:40:19] [1/INFO] [tML]: Initializing: MagicStorage v0.4.3.5
[19:40:19] [1/INFO] [tML]: Initializing: RecipeBrowser v0.8.7
[19:40:19] [1/DEBUG] [tML]: Hook On.Terraria.Player::AdjTiles() added by RecipeBrowser_0
[19:40:20] [1/INFO] [tML]: Setting up...
[19:40:20] [1/INFO] [tML]: Loading: ModLoader v0.11.7.7
[19:40:20] [1/INFO] [tML]: Loading: AnglerQuestAnnouncement v3.0.0.1
[19:40:20] [1/INFO] [tML]: Loading: BossChecklist v1.1.4.1
[19:40:20] [1/INFO] [tML]: Loading: CalamityMod v1.4.5.3
[19:40:23] [1/INFO] [tML]: Loading: CalamityEnglish v0.3
[19:40:23] [1/INFO] [tML]: Loading: CalamityModMusic v1.4.5.3
[19:40:23] [1/DEBUG] [tML]: Hook IL.Terraria.Main::UpdateAudio() modified by CalamityModMusic_0
[19:40:24] [1/INFO] [tML]: Loading: CalTEDestroyer v1.0
[19:40:24] [1/INFO] [tML]: Loading: CalValEX v6.0
[19:40:24] [1/INFO] [tML]: Loading: CMusicPlanet v1.1
[19:40:24] [1/INFO] [tML]: Loading: MagicStorage v0.4.3.5
[19:40:24] [1/INFO] [tML]: Loading: RecipeBrowser v0.8.7
[19:40:25] [1/INFO] [tML]: Adding Recipes...
[19:40:39] [Threadpool worker/INFO] [Terraria]: Loading World: fihs, IsCloud=False
[19:40:39] [Threadpool worker/INFO] [StatusText]: Resetting game objects
[19:40:51] [Threadpool worker/INFO] [StatusText]: Loading world data
[19:40:54] [Threadpool worker/INFO] [StatusText]: Settling liquids
[19:41:06] [1/INFO] [Terraria]: Running one update...
[19:41:06] [1/INFO] [tML]: Server starting with AllowVanillaClients set to False
[19:41:06] [1/DEBUG] [tML]: Fallback synchronization context assigned
[19:41:06] [Threadpool worker/DEBUG] [tML]: Assembly Resolve: Terraria, Version=1.3.5.3, Culture=neutral, PublicKeyToken=null -> Steamworks.NET, Version=9.1.0.0, Culture=neutral, PublicKeyToken=null
[19:41:18] [1/INFO] [Terraria]: Listening on port 7777
[19:41:24] [Threadpool worker/DEBUG] [tML]: Assembly Resolve: Terraria, Version=1.3.5.3, Culture=neutral, PublicKeyToken=null -> MP3Sharp, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null
[19:41:24] [Threadpool worker/DEBUG] [tML]: Assembly Resolve: Terraria, Version=1.3.5.3, Culture=neutral, PublicKeyToken=null -> NVorbis, Version=0.8.4.0, Culture=neutral, PublicKeyToken=null
[19:41:26] [Threadpool worker/INFO] [Terraria]: JIT loading finished
[19:43:59] [Threadpool worker/INFO] [Terraria]: 63.170.91.101:57054 is connecting...
[19:43:59] [1/INFO] [Terraria]: Sending Server Config AnglerQuestAnnouncement AQAConfig: {}
[19:43:59] [1/INFO] [Terraria]: Sending Server Config CMusicPlanet ModConfiguration: {}
[19:44:03] [15/ERROR] [tML]: Unhandled Exception
System.TypeLoadException: Could not resolve type with token 01000008
  at System.Threading.ThreadHelper.ThreadStart_Context (Object state) [0x00017] in mscorlib.dll
  at System.Threading.ExecutionContext.RunInternal (ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx) [0x0008d] in mscorlib.dll
  at System.Threading.ExecutionContext.Run (ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx) [0x00000] in mscorlib.dll
  at System.Threading.ExecutionContext.Run (ExecutionContext executionContext, ContextCallback callback, Object state) [0x00031] in mscorlib.dll
  at System.Threading.ThreadHelper.ThreadStart () [0x0000b] in mscorlib.dll
```
### Steps to reproduce

1. Install tModloader 
2.Install the following mods
Loading: AnglerQuestAnnouncement v3.0.0.1
Loading: BossChecklist v1.1.4.1
Loading: CalamityMod v1.4.5.3
Loading: CalamityEnglish v0.3
Loading: CalamityModMusic v1.4.5.3
Loading: CalTEDestroyer v1.0
Loading: CalValEX v6.0
Loading: CMusicPlanet v1.1
Loading: MagicStorage v0.4.3.5
Loading: RecipeBrowser v0.8.7
3. Attempt to log into the server

### Expected behavior 

Log in 

### Actual behavior

Game crashes

### Reproduction frequency

Every time

**Comment by JavidPack** [comment]
Can you narrow down the issue? If it works with 0 mods, then it's likely a problem with a specific mod and not something we can fix. Try 0 mods and keep adding mods one by one until you figure out which mod isn't working, then don't use that mod.

**Comment by qhacim** [comment]
i'm having the same problem and tried adding all my mods with no luck, so i've got no idea.

**Comment by JavidPack** [comment]
These kind of support questions are more suited for the discord chat. Anyway, the suggestion was to reduce the number of mods, not add all your mods qhacim.

**Comment by Solxanich** [comment]
Hmm, given the timing, I believe the issue here is the combination of two conflicting mods:
Calamity and CalTEDestroyer.
At beginning of October Calamity updated and wiped out all its tile entities that had issues (https://calamitymod.gamepedia.com/1.4.5.003) 
The latter mod was specifically designed as "Calamity Rust and Dust World Fixer" to destroy those removed tile entities.
TL;DR type issue would probably be cross-mod attempting to kill a tile entity that doesn't exist. Remove CalTEDestroyer.
Recommend close?
